packages/radio-player/src/search-results-switcher.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  property,
+  CSSResult,
+  TemplateResult,
+  PropertyValues,
+} from 'lit-element';
+
+@customElement('search-results-switcher')
+export default class SearchResultsSwitcher extends LitElement {
+  @property({ type: Number }) numberOfResults = 0;
+
+  @property({ type: Number }) currentResultIndex = 0;
+
+  render(): TemplateResult {
+    return html`
+      <div class="container">
+        <a @click=${this.goToPreviousResult}>&lt;</a>
```

Comment:
Are we changing the design of these with a later commit? I thought it was brought up during the demo that this resembled brackets around a data set rather than controls.